guides/tutorials/codelabs/halyard-getting-started/index.md

```diff
@@ -20,6 +20,11 @@ In our scenario, we want to create a Spinnaker instance and set it up as follows
 
 For this exercise we will be operating entirely within one GCP project, and use Google Container Engine (GKE) as our Kubernetes cluster. 
 
+{% include figure 
```

Comment:
Just curious, why not just embed the image via markdown, i.e. `![](deployment.png)?`